indico/queries/datasets.py

```diff
@@ -197,6 +231,19 @@ def __init__(self, files: List[str]):
         )
 
 
+class UploadImages(UploadDocument):
+    """
+    Upload image files stored on a local filepath to the indico platform.
+    """
+
+    def process_response(self, uploaded_files: List[dict]) -> List[str]:
+        errors = [f["error"] for f in uploaded_files if f.get("error")]
+        if errors:
+            return IndicoInputError(error="\n".join(error for error in errors),)
+        urls = [URL_PREFIX + f["path"] for f in uploaded_files]
```

Comment:
I might be misunderstanding but isn't the url prefix already on the path of these objects or are they not coming back as urls like for documents?

Reply:
No, they don't have URL prefixes coming off of rainbow `/store`. For documents, the filemeta is passed back and then doctor creates a storage object from un-prefixed url. In this case, we add the URL so that it can be identified as such for dataset upload.